src/app/asset-upload/asset-upload.service.ts

```diff
@@ -21,6 +21,13 @@ export class AssetUploadService {
     });
   }
 
+  updateIsAssetLink(isExternalAsset: boolean) {
```

Comment:
We should also update this method name to be consistent.
```suggestion
  updateIsExternalAsset(isExternalAsset: boolean) {
```